plugin/src/main/java/com/denizenscript/denizen/scripts/containers/core/ItemScriptHelper.java

```diff
@@ -629,6 +638,9 @@ public void onBrewingStandFuel(BrewingStandFuelEvent event) {
     @EventHandler(priority = EventPriority.LOW)
     public void onItemSmithing(PrepareSmithingEvent event) {
         ItemStack inputItem = event.getInventory().getItem(0);
+        if (NMSHandler.getVersion().isAtLeast(NMSVersion.v1_20)) {

```

Comment:
Should probably be a ternary on the index parameter, rather than getting item 0 and immediately replacing with item 1